### PR TITLE
Add identity section

### PR DESCRIPTION
**Pull request by mihajlija**
cc @agektmr 

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

File: src/site/_data/paths/identity.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      "pathItems": [
+        "sms-otp-form"   
+      ]
+    },
+    {
+      "title": "i18n.paths.identity.topics.build_advanced_authentication_systems",
+      "pathItems": [
+      {
+        "title": "Set up a biometric re-authentication functionality with WebAuthn",
+        "url": "https://developers.google.com/codelabs/webauthn-reauth/#0"
```
Comment (agektmr) [inline src/site/_data/paths/identity.json:34]:
```suggestion
        "url": "https://codelabs.developers.google.com/codelabs/webauthn-reauth/"
```

**Review by agektmr — APPROVED** [review]
LGTM % suggestion

**Comment by agektmr** [comment]
Can someone from web.dev eng team review this?

File: src/site/_data/i18n/paths/identity.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+title:
+  en: Identity
+  es: Identity
```
Comment (devnook) [inline src/site/_data/i18n/paths/identity.yml:3]: Please don't add untranslated strings here. It should be sufficient to have only "en" string, if others are not translated yet.

Comment (mihajlija) [reply src/site/_data/i18n/paths/identity.yml:3]: Is there a process for requesting translations?

File: src/site/_data/i18n/paths/identity.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  ja: Identity
+  ko: Identity
+  pt: Identity
+  ru: Identity
+  zh: Identity
```
Comment (mihajlija) [inline src/site/_data/i18n/paths/identity.yml:8]:
```suggestion
```

**Comment by devnook** [comment]
One more question: since the articles got moved to a new "Identity" section, do we still want them to appear in "blog"?

**Comment by mihajlija** [comment]
@devnook Removed posts from blog and fixed the translations.

**Review by devnook — APPROVED** [review]

**Pull request merged**